znvis/video/video_manager.py

```
@@ -0,0 +1,288 @@
+        # Create video writer
+        video = cv2.VideoWriter(
+            output_path.as_posix(),
+            fourcc,
+            self.frame_rate,
+            (width, height),
+        )
+        if not video.isOpened():
+            raise RuntimeError(
+                f"Failed to initialize VideoWriter for '{output_path}' "
+                f"with codec '{codec} at {width}x{height}@{self.frame_rate}fps."
+            )
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)